### DISCUSSION TITLE
Is Rust Language going to be supported by Github Copilot soon?

### DISCUSSION THREAD

**Discussion by Edu4rdSHL** [community/community#6657] [Copilot Conversations] [answered]
Hello, I hope that you all are fine. Basically the question in the topic. Thank you.

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
GitHub Copilot is not limited to any programming language, you should be able to use it with Rust just fine.

**Reply by Edu4rdSHL** [reply]
Hello, thanks for the reply. I'm a full-time Rust coding, Copilot was enabled 2 days ago on my account, and still didn't receive any suggestion from Copilot. Do you have an example or something like that? I have tried "obvious" things like `fn appendToFile(line: &str)`, `fn calculate_area(width: u64, height: u64)` and others.

**Reply by Edu4rdSHL** [reply]
I had to restart VSCode and now it's working :), thanks.
